bin/oscapd-evaluate

```diff
@@ -176,12 +176,11 @@ def cli_scan(args, config):
                                 args.xccdf_id, args.profile
                             )
                         except ProfileSuffixMatchError as e:
```

Comment:
The error message is printed when the exception is caught. This happens on the line 206-ish.